packages/@coorpacademy-player-services/src/analytics.js

```diff
@@ -1,17 +1,33 @@
+// @flow strict
+
+import type {Config, Progression, State} from '@coorpacademy/progression-engine';
+import type {Resource, ResourceType} from './types';
+
+// eslint-disable-next-line no-shadow
+declare var window: {|
+  dataLayer: Array<{
+    event: string,
+    mediaType?: string,
+    location?: string
+  }>
+|};
+
 // eslint-disable-next-line import/prefer-default-export
-export const sendViewedMediaAnalytics = (resource, location) => {
+export const sendViewedMediaAnalytics = (resource: Resource, location: string) => {
   window.dataLayer = window.dataLayer || [];
   window.dataLayer.push({
     event: 'mediaViewed',
-    mediaType: resource.type,
+    mediaType: (resource.type: ResourceType),
     location
   });
```

Comment:
This is not a good idea to use global vars like `window`. This is a browser usage, mobile doesn't have `window`

Reply:
yes, as spoken, we'll have to split services depending on usages
as of today all these services act as "web offline mocks".